CHANGES.txt

```diff
@@ -1,10 +1,10 @@
 CHANGELOG
 =========
 
-0.3.0 (unreleased)
+0.3.0 (2015-10-14)
 ------------------
 
-- Nothing changed yet.
+- Make sure requests-hawk handles requests json parameter. (#7)
```

Comment:
We're already in the requests-hawk project. consider: "make sure json parameters are handled properly"